### ISSUE TITLE
Apate doesnt show the hidden text after BDFD update.

### ISSUE THREAD

**Issue by IamNickerson** [TheGreenPig/Apate#70]
Hey,
The Apate plugin isn't showing the hidden text anymore could be BDFD update which caused it or probably discord which they changed something.

Screenshots:
![image](https://user-images.githubusercontent.com/61643704/173174219-31e97340-af82-4b5d-bb83-1855bf17d946.png)
(Doesn't Show the hidden Text)

(Some bigass console error from Betterdiscord patcher)
![image](https://user-images.githubusercontent.com/61643704/173174275-6aff51f7-b261-4f9a-9740-5342f9f778dc.png)

**Comment by SnakeBadger** [comment]
Yeah, I have the same issue as well, it would be good if it were fixed.
